lib/src/test/resources/generators/play-2-standalone-json-spec-quality.txt

```diff
@@ -489,23 +489,32 @@ package com.gilt.quality.v0.models {
     import play.api.libs.json.JsString
     import play.api.libs.json.Writes
     import play.api.libs.functional.syntax._
+    import org.joda.time.format.DateTimeFormat
+    import org.joda.time.format.DateTimeFormatterBuilder
     import com.gilt.quality.v0.models.json._
 
+    private[v0] val DateTimeFormatter =
+      new DateTimeFormatterBuilder().append(
+        org.joda.time.format.ISODateTimeFormat.dateTime.getPrinter,
+        Array(
+          DateTimeFormat.forPattern("yyyy-MM-dd'T'HH:mm:ssZZ").getParser,
+          DateTimeFormat.forPattern("yyyy-MM-dd'T'HH:mm:ss.SSSZZ").getParser
+        )
+      ).toFormatter
```

Comment:
I think we might need to call `withOffsetParsed` after `toFormatter`, otherwise the date time gets normalised to system timezone:

```
scala> val t1 = formatter.parseDateTime("2016-12-24T15:36:54+0400")
t1: org.joda.time.DateTime = 2016-12-25T00:36:54.000+13:00

scala> formatter.print(t1)
res1: String = 2016-12-25T00:36:54.000+13:00

scala> val t1 = formatter.withOffsetParsed.parseDateTime("2016-12-24T15:36:54+0400")
t1: org.joda.time.DateTime = 2016-12-24T15:36:54.000+04:00

scala> formatter.print(t1)
res2: String = 2016-12-24T15:36:54.000+04:00
```